source/databases-collections.txt

```diff
@@ -0,0 +1,275 @@
+and collections inherit them from the database. However, you can change these
+settings on your database or collection by using one of the following methods:
+
+- ``get_database()``: Gets the database with its settings inherited
+  from the client and changes the specified settings.
+- ``database.with_options()``: Gets the database and changes the specified
+  settings.
+- ``get_collection()``: Gets the collection and changes the specified settings.
+- ``collection.with_options()``: Gets the collection with its settings inherited
+  from its database and changes the specified settings.
+
+To change read or write settings with the preceding methods, call the method and
+pass in the collection or database name, and the new read preference, read
+concern, or write preference.
+
+The following example shows how to change the read preference, read concern and
+write preference of a database called ``test-database`` with the ``get_database()`` method:
+
+.. code-block:: python
+
+   client.get_database("test-database", 
+                       read_preference=ReadPreference.SECONDARY,
+                       read_concern="local",
+                       write_concern="majority")
+
+The following example shows how to change read and write settings of a
+collection called ``test-collection`` with the ``get_collection()`` method:
+
+.. code-block:: python
+
+   database.get_collection("test-collection", 
+                       read_preference=ReadPreference.SECONDARY,
+                       read_concern="local",
+                       write_concern="majority")
+
+The following example shows how to change read and write settings of a
+collection called ``test-collection`` with the ``with_options()`` method:
+
+.. code-block:: python
+
+   collection.with_options(read_preference=ReadPreference.SECONDARY,
+                           read_concern="local",
+                           write_concern="majority")
+
+.. tip::
+
+   To see the types of read preferences available in the ``ReadPreference`` enum, see the
+   `API documentation <{+api-root+}pymongo/read_preferences.html#pymongo.read_preferences.ReadPreference>`__.
+
+To learn more about the read and write settings, see the following guides in the
+MongoDB Server manual:
+
+- :manual:`Read Preference </core/read-preference/>`
+- :manual:`Read Concern </reference/read-concern/>`
+- :manual:`Write Concern </reference/write-concern/>`
+
+Tag Sets
+~~~~~~~~
+
+You can :manual:`tag </core/read-preference-tags/>` replica-set
+members according to any criteria you choose. By default, {+driver-short+} ignores tags
+when choosing a member to read from, but you can configure your read preference with
+a ``tag_sets`` parameter.
+
+The ``tag_sets`` parameter is a list of dictionaries. Each dictionary
+includes tag values that the replica-set member must match to be selected for use.
+{+driver-short+} attempts each set of tags, in order, until it finds a set of
+tags with at least one matching replica-set member.
```

Comment:
S: Cut this explanation. I was thinking about how to clarify it, but the server docs (which we already linked to) already do that. the previous paragraph mentions briefly what tags are, then the next one shows how to specify them with pymongo. i think that's sufficient.